src/components/Button.js

```diff
@@ -1,9 +1,13 @@
 import React from "react";
 
+const handelClick = () => {
+  console.log("button clicked!")
+}
+
 const Button = ({ label, onClick }) => {
```

Comment:
Der console.log funktioniert, aber die nachricht ist jetzt immer gleich für jede button instanz. stattdessen wäre es dynamischer, wenn du den prop `onClick` auf Zeile 7 als Funktion weitergibst (auf Zeile 10). Dann kannst du dem Button in `App.js` den onClick mitgeben. 

So in etwa:
`<Button label={"ok"} onClick={handleClick} />`
Das hat den Vorteil, dass du so mehrere buttons erstellen kannst aber mit unterschiedlichen Funktionen beim klicken.